### PR TITLE
[WIP] Adjust how payment fees being set on orders

### PR DESCRIPTION
**Pull request by stloyd**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Fixed tickets | #3243

### CODE REVIEW

File: src/Sylius/Component/Core/OrderProcessing/PaymentChargesProcessor.php [hunk 1]
```diff
@@ -50,24 +60,21 @@ public function __construct(RepositoryInterface $adjustmentRepository, Delegatin
...
-     * @param OrderInterface   $order
-     * @param PaymentSubjectInterface $payment
-     */
-    private function addAdjustmentIfForNotCancelled(OrderInterface $order, PaymentSubjectInterface $payment)
-    {
-        if (PaymentInterface::STATE_CANCELLED !== $payment->getState())
-        {
-            $order->addAdjustment($this->prepareAdjustmentForOrder($payment));
+            foreach ($order->getAdjustments(AdjustmentInterface::PROMOTION_ADJUSTMENT) as $adjustment) {
+                if ($payment === $this->originator->getOrigin($adjustment)) {
```
Comment (aramalipoor) [inline src/Sylius/Component/Core/OrderProcessing/PaymentChargesProcessor.php]: Shouldn't you check by `getId()` or `Comparable` interface or something?

**Comment by michalmarcinkowski** [comment]
Closing as the payment fees have been reverted (#3911).

**Comment by stloyd** [comment]
Any details why, what was wrong etc.? Saying just it was "reverted" in totally different PR it quite wrong reason for closing IMO.

**Comment by michalmarcinkowski** [comment]
@stloyd the payment fees have been removed from Sylius. The `PaymentChargesProcessor` class and all the logic for applying payment fees does not exist in Core anymore. It increased the Order processing complexity, so we decided to drop the support for this feature until we stabilize the Core logic for Order and Checkout.

**Comment by stloyd** [comment]
Ok, that much cleaner, thanks. Is there issue open about payment fees introduction etc.?

**Comment by michalmarcinkowski** [comment]
Not sure... You need to check.

**Pull request closed without merging**